pyintegration/deephaven2/time.py

```diff
@@ -0,0 +1,775 @@
+#
+#   Copyright (c) 2016-2021 Deephaven Data Labs and Patent Pending
+#
+""" This module defines functions for handling Deephaven date/time data. """
+
+import jpy
+
+from deephaven2 import DHError
+from deephaven2.constants import TimeZone
```

Comment:
I think this import suggests that `TimeZone` should be here and not in `constants`.